packages/plugin-core/package.json

```diff
@@ -170,6 +170,12 @@
         "docLink": "dendron.topic.lookup.md",
         "docPreview": "![](https://foundation-prod-assetspublic53c57cce-8cpvgjldwysl.s3-us-west-2.amazonaws.com/assets/images/schema-lookup.gif)"
       },
+      {
+        "command": "dendron.lookupSchemaV3",
```

Comment:
can we change the name to `Schema Lookup` so its consistent with the style of `Note Lookup`?

Reply:
The title of `NoteLookupCommand` is `Lookup Note` so I was going for `Lookup Schema`, but that's already taken for lookupv2's schema lookup, went with the current one.

```
{
        "command": "dendron.lookup",
        "title": "Dendron: Lookup",
        ... // this is lookup v2
},
      {
        "command": "dendron.lookupNote",
        "title": "Dendron: Lookup Note",
        ... // this is NoteLookupCommand
      },
      {
        "command": "dendron.lookupSchema",
        "title": "Dendron: Lookup Schema",
        ... // this is lookup v2 with schema flavor
      },
      {
        "command": "dendron.lookupSchemaV3",
        "title": "Dendron: Lookup Schema V3",
        ... // this is SchemaLookupCommand
      },
```

Probably not a good idea to change v2 naming, so I would go for `Schema Lookup` and change `Lookup Note` to `Note Lookup`

Reply:
ah, I see. 

in that case, feel free to delete
```
  LOOKUP_SCHEMA: {
    key: "dendron.lookupSchema",
    title: `${CMD_PREFIX} Lookup Schema`,
    group: "navigation",
    keybindings: {
      mac: "cmd+shift+L",
      key: "ctrl+shift+l",
      when: DendronContext.PLUGIN_ACTIVE,
    },
    desc: "Initiate schema lookup",
    docLink: "dendron.topic.lookup.md",
    docPreview:
      "![](https://foundation-prod-assetspublic53c57cce-8cpvgjldwysl.s3-us-west-2.amazonaws.com/assets/images/schema-lookup.gif)",
  },
```

and update lookup schema to have the new keyboard shortcut. the plan is to replace it and schema lookup should be at full functionality now.